src.ts/utils/key_pair.ts

```diff
@@ -86,6 +86,20 @@ export abstract class KeyPair {
             throw new Error('Invalid encoded key format, must be <curve>:<encoded key>');
         }
     }
+    
+    static fromSeed(seedStr: string): KeyPair {
+        const seed = new Uint8Array(32);
+        for (let i = 0; i < 32; i++) {
```

Comment:
it seems like using only 32 characters from a phrase which supposedly contains common words doesn't provide nearly as much entropy as needed for strong 256-bit key. It's basically only enough for about 6 word seed phrase. I suggest we abandon this scheme altogether (i.e. not use it in Rust either).

Reply:
I think we use this in rust mostly for testing purposes.